process-dataset/process-single-dataset.js

```diff
@@ -45,14 +47,17 @@ const processSingleDataset = async (id, datasetJson, shouldSkipFileInfoUpload, m
     // 6. upload cell line subtotals
     await uploadCellCountsPerCellLine(TEMP_FOLDER, firebaseHandler);
     // 7. upload json to aws
-    const awsLocation = await uploadFileToS3(firebaseHandler.id, TEMP_FOLDER);
-    // 8. upload card image
+    const awsLocation = await uploadFeaturesFileToS3(firebaseHandler.id, TEMP_FOLDER);
+    // 8. upload viewer settings json to aws
+    const awsViewerSettingsLocation = await uploadFileToS3(firebaseHandler.id, datasetReadFolder, fileNames.viewerSettingsData);
+    // 9. upload card image
     const awsImageLocation = await uploadDatasetImage(firebaseHandler, datasetReadFolder, datasetJson.image);
-    // 9. update dataset manifest with location for data
+    // 10. update dataset manifest with location for data
     const updateToManifest = {
         ...featureDefRef,
         ...fileInfoLocation,
         ...awsLocation,
+        ...{viewerSettingsPath: awsViewerSettingsLocation}
```

Comment:
I don't know if this is a matter of preference or what but I think it's cleaner and equivalent to just do this?
```javascript
        ...featureDefRef,
        ...fileInfoLocation,
        ...awsLocation,
        viewerSettingsPath: awsViewerSettingsLocation
```

Reply:
works for me, I'll update it